packages/subgraph/schema.graphql

```diff
@@ -399,27 +399,27 @@ type AdminPlugin implements IPluginInstallation @entity {
   pluginAddress: Bytes!
 
   proposals: [AdminProposal!]! @derivedFrom(field: "plugin")
-  adminstrators: [AdminstratorAdminPlugin!]! @derivedFrom(field: "plugin")
+  administrators: [AdministratorAdminPlugin!]! @derivedFrom(field: "plugin")
```

Comment:
can't this be `Administrator` entity ? Seems like we're bringing one extra entity for no reason.

Reply:
No because the Administrator entity doesn't get deleted and we wanna have a list of active admins for each DAO

Reply:
so, administrators: [AdministratorAdminPlugin!] store currently, active admins ?

Reply:
`AdministratorAdminPlugin` naming seems confusing then.

Reply:
AdministratorAdminPlugin naming seems confusing then.

Reply:
it is a `many:many` connection between Plugins and Administrators. I also left it as it is to not make any more work to the sdk and the frontend

Reply:
I get it, but naming is misleadin. which one are current administrators, which one are all(current + removed).

Reply:
agreed. changing the naming impacts the sdk and the frontend and with the current timeline I would change more. We can refactor it as soon was we move the plugins out of the core